client/src/modules/Home/Home.module.css

```diff
@@ -189,6 +189,6 @@
 }
 
 .bgnightspring {
-  background-image: url('../../assets/Home/bg_night_spring.svg'),
+  background-image: url('../../assets/Home/bg_night_summer.svg'),
```

Comment:
If we're going to be using this file for both summer and spring nighttime views, it may be helpful to update the file name.

Reply:
Ye, I didn't download the spring version cause I was too lazy but I updated it now!